### PR TITLE
Update project configuration

### PR DESCRIPTION
**Pull request by Eomm**
Closing #15 
Closing #6

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
-- master
-
 pool:
-  vmImage: 'Ubuntu-16.04'
+  vmImage: 'ubuntu-16.04'
+
+strategy:
+  matrix:
+    # node_8_x:
+    #   node_version: 8.x
```
Comment (mcollina) [inline azure-pipelines.yml]: remove these lines, if you are not targeting node 8. I’m ok with that.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
-# https://aka.ms/yaml
-
-trigger:
-- master
-
 pool:
-  vmImage: 'Ubuntu-16.04'
+  vmImage: 'ubuntu-16.04'
```
Comment (mcollina) [inline azure-pipelines.yml]: I would prefer if we also tested on Windows and maybe Mac.

**Comment by Eomm** [comment]
Done

if needed by users, I will check why Node 8 isn't working in future

File: azure-pipelines.yml [hunk 3]
```diff
@@ -1,19 +1,35 @@
...
+      node_version: 10.x
+    linux_node_11:
+      imageName: 'ubuntu-16.04'
+      node_version: 11.x
+    mac_node_11:
+      imageName: 'macos-10.13'
+      node_version: 11.x
+    windows_node_11:
+      imageName: 'vs2017-win2016'
+      node_version: 11.x
```
Comment (mcollina) [inline azure-pipelines.yml]: I would remove Node 11 and add Node 12 instead.

**Pull request merged**